generators/entity-server/templates/src/main/java/package/domain/Entity.java.ejs

```diff
@@ -158,9 +159,14 @@ public class <%= entityClass %> implements Serializable {
     const fieldNameUnderscored = fields[idx].fieldNameUnderscored;
     const fieldNameAsDatabaseColumn = fields[idx].fieldNameAsDatabaseColumn;
     if (fieldValidate === true) {
-        if (fieldValidate === true && fieldValidateRules.includes('required')) {
-            required = true;
-        } _%>
+        if (fieldValidate === true) {
```

Comment:
Condition already tested at line 161 (yes, the original code was already testing twice).
Please change the start of the block to 
```
if (fieldValidate) {
    required = fieldValidateRules.includes('required')`;
    unique = fieldValidateRules.includes('unique')`;
```